server/src/main/java/io/druid/segment/realtime/RealtimeMetricsMonitor.java

```diff
@@ -96,6 +96,9 @@ public boolean doMonitor(ServiceEmitter emitter)
       emitter.emit(builder.build("ingest/merge/time", metrics.mergeTimeMillis() - previous.mergeTimeMillis()));
       emitter.emit(builder.build("ingest/merge/cpu", metrics.mergeCpuTime() - previous.mergeCpuTime()));
       emitter.emit(builder.build("ingest/handoff/count", metrics.handOffCount() - previous.handOffCount()));
+      emitter.emit(builder.build("ingest/sink/count", metrics.sinkCount()));
+      emitter.emit(builder.build("ingest/events/messageGap", metrics.messageGap()));
+      emitter.emit(builder.build("ingest/events/sourceGap", metrics.sourceGap()));
```

Comment:
it looks like, this one will not be correctly reported by druid's realtime ingestion. am i correct?
is it possible to not have it here but only in your custom extension/code that has the ability to report this?
this metric will only confuse to general druid user and they need to completely ignore this one.